test/edip/settings_test.exs

```diff
@@ -0,0 +1,55 @@
+defmodule SettingsTest do
+  use ExUnit.Case
+
+  alias Edip.Options
+
+  test "package name is converted to environment var spec string" do
+    pkg_options =
+      %Options.Package{name: "pkg_name"}
+      |> Edip.Settings.from_package_options
+
+   assert String.contains?(pkg_options, "RELEASE_NAME=pkg_name")
+  end
+
+  test "tag name is converted to environment var spec string" do
+    pkg_options =
+      %Options.Package{tag: "taggar"}
+    |> Edip.Settings.from_package_options
+
+    assert String.contains?(pkg_options, "RELEASE_TAG=taggar")
+  end
+
+  test "package prefix is converted to environment var spec string" do
+    pkg_options =
+      %Options.Package{prefix: "prefixer"}
+    |> Edip.Settings.from_package_options
+
+    assert String.contains?(pkg_options, "RELEASE_PREFIX=prefixer")
+  end
+
+  test "basic mapping is formatted for docker run command" do
+    mapping_options =
+      [%Options.Mapping{from: "/from/vol", to: "/to/vol"}]
+    |> Edip.Settings.from_mapping_options
+
+    assert mapping_options == "-v \"/from/vol:/to/vol\""
```

Comment:
If you want to get rid of the backslash escaping for double quotes you can use the `~s()` instead:

```elixir
~s(-v "/from/vol:/to/vol")
```